HonzaBotner.sln

```diff
@@ -10,7 +10,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		Directory.Build.props = Directory.Build.props
 		CONTRIBUTING.md = CONTRIBUTING.md
 		Packages.props = Packages.props
-		renovate.json = renovate.json
```

Comment:
Why this? I have added it intentionally.

Reply:
I moved it to .github so that it doesn't contribute to the mess we already have in root :smile:

Reply:
And somehow this was deleted automatically after me moving it.